wiki/en/Running-a-Server.md

```diff
@@ -274,7 +274,7 @@ For macOS, start a Terminal window and run Jamulus with the desired options like
 
 ## Running an Unregistered Server
 
-It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
+It is highly recommended to test your Server on **Registered Directory** first so as to narrow down any subsequent problems in unregistered mode.
```

Comment:
There's no such thing as a "Registered Directory".  "Directory" is qualified by either "Public" or "Custom" depending on whether it's built in or manually entered by the user.  Here the intent is "Public" (as it makes things simpler).

Reply:
Yes. Just spotted that after I did the PR so have corrected.

Reply:
New version is nice.